src/main/kotlin/models/Task.kt

```diff
@@ -0,0 +1,22 @@
+package models
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Example(val input: String = "", val output: String = "")
```

Comment:
I am not sure that you actually need those default values. In what situations do you create it with all defaults?